### PR TITLE
15291 hotfix hide sole proprietor change button v2

### PR DESCRIPTION
**Pull request by jonathan-longe**
*Issue #:* /bcgov/entity#15291

HOTFIX based based on commit 1df7fff098c436b4ac64f4cec54dbf17fdd9e900  This PR is identical to #444 except it's based on release branch `release-3.9.15`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue [hunk 1]
```diff
@@ -283,7 +283,7 @@
 
               <!-- org-person we haven't touched: -->
               <div v-else class="actions mr-4">
-                <span class="edit-action">
+                <span class="edit-action" v-if="hideChangeButtonForSoleProps">
```
Comment (severinbeauvais) [inline src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue:286]: Should be "not".

And getter logic (and comment) should be the opposite.

**Review by severinbeauvais — COMMENTED** [review]
Please update the app version.

**Comment by jonathan-longe** [comment]
Done.  Thanks @severinbeauvais for all your help

**Review by severinbeauvais — APPROVED** [review]
LGTM

Much better getter comment :)

I re-ran the failed GH check. It's OK now. Merge away.

**Pull request merged**